core/cmd/local_client.go

```diff
@@ -214,65 +215,47 @@ func (cli *Client) runNode(c *clipkg.Context) error {
 		}
 
 		for _, ch := range evmChainSet.Chains() {
-			skey, sexisted, fkey, fexisted, err2 := app.GetKeyStore().Eth().EnsureKeys(ch.ID())
+			err2 := app.GetKeyStore().Eth().EnsureKeys(ch.ID())
 			if err2 != nil {
 				return errors.Wrap(err2, "failed to ensure keystore keys")
 			}
-			if !fexisted {
-				lggr.Infow("New funding address created", "address", fkey.Address.Hex(), "evmChainID", ch.ID())
-			}
-			if !sexisted {
-				lggr.Infow("New sending address created", "address", skey.Address.Hex(), "evmChainID", ch.ID())
-			}
 		}
 	}
 
 	if cli.Config.FeatureOffchainReporting() {
-		ocrKey, didExist, err2 := app.GetKeyStore().OCR().EnsureKey()
-		if err2 != nil {
+		err2 := app.GetKeyStore().OCR().EnsureKey()
+		if err2 != nil && !errors.Is(err, keystore.ErrOCRKeySeeded) {
```

Comment:
Seems like you are checking err variable instead of err2 in "!errors.Is(err, ...".

The same mistake is in few more places below.

Reply:
If you return nil from these methods if key already existed, you can get rid of this check entirely.